image_explorer/image_explorer.py

```diff
@@ -354,7 +355,10 @@ def _inner_content(self, tag, absolute_urls=False):
         Helper met
         """
         if tag is not None:
-            tag_content = u''.join(html.tostring(e) for e in tag)
+            if six.PY3:
+                tag_content = b''.join(html.tostring(e) for e in tag).decode('utf-8')
```

Comment:
this can be simplified to `''.join(html.tostring(e, encoding=str) for e in tag)`